dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -214,6 +222,236 @@
   puts "Creating applications took #{time_end - time_start} seconds"
 end
 
+And(/^I am viewing a workshop with fake survey results$/) do
+  require_rails_env
+
+  workshop = FactoryGirl.create :pd_ended_workshop, :local_summer_workshop,
+    organizer: FactoryGirl.create(:workshop_organizer, email: "test_organizer#{SecureRandom.hex}@code.org"),
+    num_sessions: 5, enrolled_and_attending_users: 10,
+    facilitators: [
+      (FactoryGirl.create :facilitator, email: "test_facilitator#{SecureRandom.hex}@code.org", name: 'F1'),
+      (FactoryGirl.create :facilitator, email: "test_facilitator#{SecureRandom.hex}@code.org", name: 'F2')
+    ]
+
+  begin
+    Pd::SurveyQuestion.find_or_create_by!(form_id: CDO.jotform_forms['local']['day_0'],
+      questions: [
+        {
+          id: 1,
+          name: 'matrix',
+          type: 'matrix',
+          text: 'How much do you agree / disagree with these statements on teaching CS?',
+          options: [
+            'Strongly Disagree',
+            'Disagree',
+            'Slightly Disagree',
+            'Neutral',
+            'Slightly Agree',
+            'Agree',
+            'Strongly Agree'
+          ],
+          sub_questions: [
+            'I like computer science',
+            'People should learn computer science',
+            'I feel like I can teach computer science'
+          ],
+          order: 1
+        },
+        {
+          id: 2,
+          name: 'scale',
+          type: 'scale',
+          text: 'How pumped are you to teach CS?',
+          options: ['Not at all pumped', 'Super pumped'],
+          values: (1..5).to_a,
+          order: 2
+        },
+        {
+          id: 3,
+          name: 'radio',
+          type: 'radio',
+          text: 'How much CS experience do you have?',
+          order: 3,
+          options: [
+            'None',
+            'Some basic messing around',
+            'Formal education',
+            'I am l33t h4xx0r'
+          ]
+        },
+        {
+          id: 4,
+          name: 'textarea',
+          type: 'textarea',
+          text: 'What inspired you to teach computer science?',
+          order: 4
+        },
+        {
+          id: 5,
+          name: 'userId',
+          text: 'userId',
+          type: 'textarea',
+          order: 5,
+          hidden: true
+        },
+        {
+          id: 6,
+          name: 'workshopId',
+          text: 'workshopId',
+          type: 'textarea',
+          order: 6,
+          hidden: true
+        }
+      ].to_json
+    )
+  rescue => e
+    puts "Unable to create SurveyQuestions. If you are running this locally, please make
+    sure that you have overridden jotform_forms in your locals.yml"
+    raise e
+  end
```

Comment:
Why rescue here but not the subsequent creates? Might it be better / simpler to just rescue from the whole method? Or explicitly check the expected CDO values. 

Should we include this message in the newly raised error instead of dumping to console?

Reply:
I wanted to do this create because failure to create the Jotform responses is unrelated to this problem. This warning message is only appropriate when there is a failure to create survey questions

Reply:
I don't follow. Isn't line 313 also creating survey questions?